rskj-core/src/main/java/co/rsk/core/bc/BlockRelayValidatorImpl.java

```diff
@@ -66,7 +71,12 @@ public boolean isValid(@Nonnull Block block) {
         }
 
         Block parent = getParent(block);
-        return parent != null && blockParentValidator.isValid(block.getHeader(), parent);
+        if (parent == null) {
+            logger.error("Missing parent for block {}, cannot relay it", block.getNumber());
```

Comment:
minor thing.. responsibility of this class is to validate blocks, not to relay them. So we could just say "Invalid block because of missing parent block" or something alike